clever/api_client.py

```diff
@@ -121,7 +121,7 @@ def __call_api(self, resource_path, method,
             for k, v in path_params:
                 # specified safe chars, encode everything
                 resource_path = resource_path.replace(
-                    '{%s}' % k, quote(str(v), safe=config.safe_chars_for_path_param))
```

Comment:
this isn't clever-specific, right? i wonder if we should try to fix this upstream?

Reply:
I'm not sure this is even a real thing... But it existed before so I'm keeping up the behavior